php_version_resolver_test.go

```diff
@@ -0,0 +1,181 @@
+package composer_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/composer"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+func testPhpVersionResolver(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		workingDir string
+
+		phpVersionResolver composer.PhpVersionResolver
+	)
+
+	it.Before(func() {
+		var err error
+		workingDir, err = os.MkdirTemp("", "working-dir")
+		Expect(err).NotTo(HaveOccurred())
+
+		phpVersionResolver = composer.NewPhpVersionResolver()
+	})
+
+	it.After(func() {
+		Expect(os.RemoveAll(workingDir)).To(Succeed())
+	})
+
+	context("when both composer.lock and composer.json contain php versions", func() {
+		it.Before(func() {
+			Expect(ioutil.WriteFile(filepath.Join(workingDir, "composer.json"), []byte(`{
+   "require": {
+	   "php-64bit": "php-64bit.version.from-composer-json",
+	   "php": "php-32bit.version.from-composer-json"
+   }
+}`), os.ModePerm)).NotTo(HaveOccurred())
+		})
+
+		context("when composer.lock contains the 64bit php version", func() {
+			it.Before(func() {
+				Expect(ioutil.WriteFile(filepath.Join(workingDir, "composer.lock"), []byte(`{
+ "platform": {
+   "php-64bit": "php-64bit.version.from-composer-lock"
+ }
+}`), os.ModePerm)).NotTo(HaveOccurred())
+			})
+
+			it(`requires "php" with version metadata`, func() {
```

Comment:
```suggestion
			it(`requires "php" with version metadata from the composer.lock`, func() {
```
same goes for the other tests

Reply:
done